src/Sylius/Bundle/AdminBundle/templates/product_variant/form/sections/channel_pricing/accordion.html.twig

```diff
@@ -0,0 +1,44 @@
+{% from '@SyliusAdmin/shared/helper/channel_pricings.html.twig' import channel_pricings %}
+
+{% set product_variant = hookable_metadata.context.form.vars.data %}
+{% set body %}
+    {% verbatim %}
+        <div class="row">
+            <div class="col-12 col-md-6">
+                {% set is_channel_pricing_invalid = channel_pricing_form.vars.errors.form.parent.errors is not empty %}
+
+                {% if is_channel_pricing_invalid %}
+                    <div class="alert alert-danger" role="alert">
+                        <div class="d-flex">
+                            <div>
+                                {{ ux_icon('tabler:exclamation-circle') }}
+                            </div>
+                            <div>
+                                {% set first_error = channel_pricing_form.vars.errors.form.parent.errors|first %}
+                                {{ first_error.messageTemplate|trans({}, 'validators') }}
+                            </div>
+                        </div>
+                    </div>
+                {% endif %}
+                {% if channel_code not in product_variant.product.channels|map(channel => channel.code) %}
+                    <div class="alert alert-info">
+                        {{ 'sylius.ui.product.product_not_active_in_channel'|trans }}
+                    </div>
+                {% endif %}
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm">
+                {{ form_row(channel_pricing_form.price) }}
+            </div>
+            <div class="col-sm">
+                {{ form_row(channel_pricing_form.originalPrice) }}
+            </div>
+            <div class="col-sm">
+                {{ form_row(channel_pricing_form.minimumPrice) }}
+            </div>
```

Comment:
Maybe also split these into separate hooks?
As it is I need to overwrite this entire file to add a new price type or any other field to the `ChannelPricing` form.